template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/RazorPagesWeb-CSharp/Data/Migrations/ApplicationDbContextModelSnapshot.cs

```diff
@@ -0,0 +1,245 @@
+﻿using System;
+#if (UseLocalDB)
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+#endif
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Metadata;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Company.WebApplication1.Data.Migrations
+{
+    [DbContext(typeof(ApplicationDbContext))]
+    partial class ApplicationDbContextModelSnapshot : ModelSnapshot
+    {
+        protected override void BuildModel(ModelBuilder modelBuilder)
+        {
+            modelBuilder
+#if (UseLocalDB)
+                .HasAnnotation("ProductVersion", "1.0.0-rc3")
```

Comment:
I assume most of the model code has been brought over from the StarterWeb template, if that is the case, should we update the the versions in the migrations to reflect the current version - It doesn't look like the migration was actually run to refresh the code in quite some time...